src/JustEat.StatsD/StatsDPublisher.cs

```diff
@@ -23,7 +40,9 @@ public StatsDPublisher(StatsDConfiguration configuration)
             }
 
             _formatter = new StatsDMessageFormatter(configuration.Culture, configuration.Prefix);
-            _transport = new StatsDUdpTransport(configuration.Host, configuration.Port, null);
+
+            var endpointSource = EndpointParser.MakeEndPointSource(configuration.Host, configuration.Port, StatsDConfiguration.DefaultDnsLookupInterval);
```

Comment:
Yeah, we could put an instance property on StatsDConfiguration with that as the default value.